support-frontend/assets/components/csr/csrMode.js

```diff
@@ -0,0 +1,17 @@
+
+const domains = ["https://gnmtouchpoint.my.salesforce.com", "https://gnmtouchpoint--dev1--c.cs88.visual.force.com"]
```

Comment:
Woudn't there be 3 SF domains overall?

Reply:
I was trying to figure out whether UAT would be useful, the interaction with test users could be complex. I'll revisit this when I do the rest of the work to integrate with the checkout, this is mostly a proof of concept for now.